From e439eba103b8d9bd079d4f8f63fc724cdaebca50 Mon Sep 17 00:00:00 2001
From: Andreas Schwab <schwab@suse.de>
Date: Thu, 11 Sep 2014 12:02:29 +0200
Subject: [PATCH 03/10] KVM: ARM: Add "arm,gic-400" compatible support
Patch-mainline: No
References: bnc#902632

Some device trees specify their GIC as compatible "arm,gic-400". The code
supports those just fine, we were only missing the compatible to make it
work.

Signed-off-by: Alexander Graf <agraf@suse.de>

---
 virt/kvm/arm/vgic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/virt/kvm/arm/vgic.c b/virt/kvm/arm/vgic.c
index 3aaca49..7613dea 100644
--- a/virt/kvm/arm/vgic.c
+++ b/virt/kvm/arm/vgic.c
@@ -2428,6 +2428,7 @@ static struct notifier_block vgic_cpu_nb = {
 
 static const struct of_device_id vgic_ids[] = {
 	{ .compatible = "arm,cortex-a15-gic", .data = vgic_v2_probe, },
+	{ .compatible = "arm,gic-400", .data = vgic_v2_probe, },
 	{ .compatible = "arm,gic-v3", .data = vgic_v3_probe, },
 	{},
 };
-- 
2.1.0

